transcribe/aws.py

```diff
@@ -34,8 +34,8 @@ def run(bags_dir, output_dir):
 
         result = utils.compare_transcripts(reference, transcription["transcript"])
         result["language"] = transcription["language"]
-        result["file"] = file
-        result["duration"] = duration
+        result["file"] = os.path.basename(file)
+        result["runtime"] = runtime
```

Comment:
I was having trouble remembering whether duration was the length of the media or the length of the processing.